### DISCUSSION TITLE
How to resolve the DepartmentId (HubSiteId) to the Hub Site Name in the search results?

### DISCUSSION THREAD

**Discussion by mozilla0** [microsoft-search/pnp-modern-search#2819] [Q&A] [answered]
I try to resolve the HubSiteId (DepartmentId) to the specific Name (instead of the Id). Is there a way without making a custom modern search webpart?

![PropertyBag_RelatedHubSiteIds](https://user-images.githubusercontent.com/17600172/222731152-3a6659bb-9455-41e7-93b0-8f82c04efc80.png)
![Resolved_Name_to_HubId](https://user-images.githubusercontent.com/17600172/222731157-a53a3e8e-1d40-4706-9c57-2555a56b5741.png)
![SearchResult](https://user-images.githubusercontent.com/17600172/222731161-91fe8434-bc23-4db4-a6ec-b9b47881ab71.png)

**Comment by wobba** [comment, marked as answer]
Unfortunately not. The hub name is not part of the results. So you'd need an extra API call to load it.
